### ISSUE TITLE
[Feature Request] Please add Energy meter for Dryer LG RH90V5AV6Q

### ISSUE THREAD

**Issue by VivantSenior** [ollo69/ha-smartthinq-sensors#290]
**Is your feature request related to a problem? Please describe.**
Feature request is not related to any issue.

**Describe the solution you'd like**
A new sensors should be visible in the Home Assistant:
- Current power usage (W) - if possible,
- Todays power usage (kWh) - if possible,
- Total power usage (kWh) - if possible,
- Start date of the total power usage measurement (date + time).

**Describe alternatives you've considered**
Installing a separate device (smart plug) to monitor power usage.

**Additional context**
Please see logs attached: [Logs Dryer LG RH90V5AV6Q.zip](https://github.com/ollo69/ha-smartthinq-sensors/files/8055399/Logs.Dryer.LG.RH90V5AV6Q.zip)

**Comment by Wingeddonkey** [comment]
Agreed on this one, came here looking for details on this. I've been using the integration for about a year now. Currently doing a deep dive into energy monitoring/auditing. The LG app logs energy (without a useful graph/units in either axis/export), it is not useful. Hoping to have a good export of that data to HA Energy and the like. 

Hoping someone smarter than me could grab the LG part of this and at it to an energy entity.
